s3repo/package.py

```diff
@@ -0,0 +1,18 @@
+"""Description of the uploaded / downloaded package."""
+
+class Package:
+    """Description of the uploaded / downloaded package."""
+
+    def __init__(self):
+        self.dist = ''
+        self.version = ''
+        # This parameter refers to the version of the tarantool.
+        self.repo = ''
+        # Repos type can be live / release / staging ...
+        self.repos_type = ''
+        # Files to upload.
+        self.files = {}
```

Comment:
> Exact formats are unclear, I would add a few example for each field.
> 
> 1. `dist` is just distro name, like `ubuntu`? Seems no, there is no `dist_version`. So, `ubuntu/focal`? Or `ubuntu:focal`?

It is a distribution name.

> 2. `version` is a free form, inherited from a package?

This is a distribution version. It's now called `dist_version`.

> 3. `repo` is `1.10`, `2.1` and so on?

Yep.(tarantool version)

> 4. `repos_type` is a kind of list (it is plural, right)? Update: I see now, you meant a type for several repositories, because a package may be deployed to several repositories at once. However it is not package-level detail. I would just say `repo_type`, it fits good here.

It's now called `repo_kind`.
`Package` describes the package files and where they should be uploaded.

> 5. `files` is a dictionary in fact. However, okay, I see it just below.

Yep.